resources/assets/js/app/instances/paginateableCollectionView.js

```diff
@@ -0,0 +1,60 @@
+var Marionette = require('backbone.marionette');
+
+module.exports = Marionette.CollectionView.extend({
+
+    initialize: function () {
+
+        $(window).on('scroll', this.fetchPage.bind(this));
+
+    },
+    onDestroy: function(){
+        $(window).off('scroll', this.fetchPage.bind(this));
```

Comment:
remove callback, and leave just $(window).off('scroll');
with callback this event still present